app/src/main/java/com/example/githubapp/user/ProfileFragment.kt

```diff
@@ -0,0 +1,42 @@
+package com.example.githubapp.user
+
+import android.content.Context
+import android.os.Bundle
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModel
+import com.example.githubapp.MyApplication
+import com.example.githubapp.R
+import com.example.githubapp.databinding.FragmentProfileBinding
+import javax.inject.Inject
+
+class ProfileFragment() : Fragment() {
+    @Inject
+    lateinit var viewModel: ProfileViewModel
```

Comment:
In the current implementation ProfileViewModel will be recreated every time the fragment is attached to activity.